### PR TITLE
Add Server rendering to react project

### PR DESCRIPTION
**Pull request by bandax**
I have this issue: 

ReferenceError: window is not defined
    at eval (webpack:///./src/components/App.tsx?:34:1)
    at Object../src/components/App.tsx (C:\Users\Luis_Banda\Documents\React\Mentoring\Movies.ReactJS\public\js\serverRenderer.js:122:1)
    at __webpack_require__ (C:\Users\Luis_Banda\Documents\React\Mentoring\Movies.ReactJS\public\js\serverRenderer.js:21:30)
    at eval (webpack:///./src/serverRenderer.tsx?:9:13)
    at Object../src/serverRenderer.tsx (C:\Users\Luis_Banda\Documents\React\Mentoring\Movies.ReactJS\public\js\serverRenderer.js:312:1)
    at __webpack_require__ (C:\Users\Luis_Banda\Documents\React\Mentoring\Movies.ReactJS\public\js\serverRenderer.js:21:30)
    at C:\Users\Luis_Banda\Documents\React\Mentoring\Movies.ReactJS\public\js\serverRenderer.js:85:18
    at Object.<anonymous> (C:\Users\Luis_Banda\Documents\React\Mentoring\Movies.ReactJS\public\js\serverRenderer.js:88:10)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at requireFromString (C:\Users\Luis_Banda\Documents\React\Mentoring\Movies.ReactJS\node_modules\require-from-string\index.js:28:4)

### CODE REVIEW

File: src/client.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  }
+}
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+declare global {
+  interface Window {
+    PRELOADED_STATE: any;
+  }
+}
```
Comment (endiny) [inline src/client.tsx:23]: I think you can join it with another `interface Window`

File: src/components/AddMovie/AddMovie.tsx [hunk 2]
```diff
@@ -58,13 +58,13 @@ const AddMovie: React.FunctionComponent<Props> = ({
...
       </div>
-      <AddMovieWindow
+      {/* <AddMovieWindow
         clasificationMovies={movieTypes}
         showModal={showAddMovieModal}
         movie={movie}
         onShowAddMovieWindow={handleShowAddMovieWindow}
         onAddMovieSubmit={handleSubmitAddMovie}
-      />
+      /> */}
```
Comment (endiny) [inline src/components/AddMovie/AddMovie.tsx:67]: Please don't keep commented code in VCS. It can be retrieved in previous commits

**Review by endiny — CHANGES_REQUESTED** [review]

File: src/client.tsx (hunk 1 above)
Comment (bandax) [reply src/client.tsx:23]: you mean change this to its own file?

Comment (endiny) [reply src/client.tsx:23]: you have two `declare global`. Merge them onto one statement\
```
declare global {
  interface Window {
  __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose;
    PRELOADED_STATE: any;
  }
}

**Pull request merged**